src/ae.c

```
@@ -278,14 +279,33 @@ long long aeCreateTimeEvent(aeEventLoop *eventLoop,
     eventLoop->timeEventHead = te;
+    if (eventLoop->earliest_timer_when == 0 || te->when < eventLoop->earliest_timer_when) {
+        eventLoop->earliest_timer_when = te->when;
+    }
     return id;
 }
 
+
+static void aeRecomputeEarliestTimer(aeEventLoop *eventLoop) {
+    aeTimeEvent *te = eventLoop->timeEventHead;
+    monotime earliest = 0;
+    while (te) {
```

Cppcheck (2.21.0)

[error] `#error` "Undefined or invalid BYTE_ORDER"

(preprocessorErrorDirective)